### DISCUSSION TITLE
Github pages custom domain says already in use, but it's not

### DISCUSSION THREAD

**Discussion by dev-yashmathur** [community/community#117239] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have recently taken a domain name, and had connected it to a github repository. However, I have since deleted that repository, and am trying to set the domain name as the URL for another repository, but it shows the error that it's already in use. I have checked, and none of my other repositories use it. Also, i have already verified the domain name for my page.

I have raised a support ticket over a week ago, but haven't gotten a response yet.  Wondering if anyone else knows any solutions for it?

I have found a similar problem by another user back in 2021 here [23569 ](https://github.com/orgs/community/discussions/23569) where the issue was resolved by the staff members manually releasing the domain at their end.

Is something similar possible here as well?

Thanks!

**Comment by AlexanderFRT** [comment]
Since you didn't actually remove the domain from the old repository before deleting it, your only hope is the GitHub Support. The chances of staff members actively checking community discussions like they have in the past seem very slim, community discussions have been plagued by spambots every day lately, and it doesn't seem like they're doing anything to stop it.

The best thing you can do is wait a little longer with your current ticket until they respond or submit an additional new ticket in another category to see if the response is faster.

**Comment by jfullstackdev** [comment]
Copilot suggests things you may find helpful, if you still have questions feel free to ask

---

The issue you're experiencing might be due to DNS caching or GitHub's internal caching. Here are a few steps you can try:

1. **Clear DNS cache**: Sometimes, DNS servers cache records for a period of time, this could be causing the issue. You can wait for the cache to clear (this could take up to 24-48 hours) or try clearing your local DNS cache.

2. **Check GitHub's domain settings**: Go to your GitHub Pages repository settings and ensure that the custom domain field is empty. If it's not, clear it and save the changes.

3. **Check your domain provider's settings**: Ensure that your domain's DNS settings are correctly configured to point to GitHub Pages. You should have a CNAME record pointing to your GitHub Pages account (i.e., `username.github.io`).

4. **Re-add the domain to the new repository**: Try adding the domain to your new repository again. Make sure to include the domain in the `CNAME` file in your repository and in the repository settings under the GitHub Pages section.

5. **Contact GitHub Support**: If none of the above steps work, it might be an issue on GitHub's end. You can contact GitHub Support for further assistance.

Remember, changes to DNS records can take up to 24-48 hours to propagate through the internet, so you might not see the changes immediately.

**Comment by dev-yashmathur** [comment]
@AlexanderFRT @jdevstatic 
Thank you both for your suggestions, however none of it helped. Almost 2 weeks and no reply from github's support team.

However I did figure out an approach to get it to work. Here are the steps I followed, as future reference if anyone else encounters this issue.

Step 1: Remove the verification records for your github profile
Step 2: Associate your custom domain with a different github account (Maybe a friend's)
Step 3: The boring part, wait for a day or (upto a week as per their documents), for github to release the domain from any other user
Step 4: Remove the custom domain from this temp user, and move it back to yours.
Step 5: Enjoy your domain once more

Bonus: Step 6: Don't forget to unlink the domain again before deleting the repo :)
